pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -634,8 +635,19 @@ def initialize(self):
         # setup spoke state based on data from the Subscription DBus module
         self._update_spoke_state()
 
+        # start the rest of spoke initialization which might take some time
+        # (mainly due to waiting for various initialization threads to finish)
+        # in a separate thread
+        threadMgr.add(AnacondaThread(name=constants.THREAD_SUBSCRIPTION_SPOKE_INIT,
```

Comment:
Isn't `constants.` redundant here? You refer to all the other threads directly by constant name...

Reply:
Indeed, this was a copy paste typo & has been fixed.